audreycode6/L2_PY101_LS
# Modified calculator.py to accept float input

# Description
Modified calculator.py to accept float input rather than just ints. Added function get_number_input() to calculator.py. Cleaned up code for calculator.py /calc_messages.json.
_--Note--_ these changes were done to the most recent version of calculator.py / calc_messages.json from internationalize_calc branch which has not been merged into main (waiting for PR's to be reviewed before merging).
_--Note--_ I thought about your advice to let users pick their default language but kinda going in rabbit hole and want to proceed with my courses, so skipping for now.

# Changes
- Altered all int constructs to float constructs. Users can now input floating point numbers.
- Updated 'num_errors' value from calc_messages.json to not reference needing integers only. I.e original: "Error: That isn't a valid number, expecting an integer!" VS updated: "Error: That isn't a valid number!"
- Cleaned up code for num1 and num2 input: using function get_number_input(), connected it to use  prompt() so that it works with translations.
- Added a call for calc_again() after user receives division by zero error so that they have option to continue calculations rather then ending program 


# QA Instructions
1. run calculator.py with variety of inputs for numbers: float, int/float, non-number -ensure works correctly.
2. Test calculator.py works using different languages as default -ensure that num1 and num2 translate properly.
